### ISSUE TITLE
issue while updating students profile picture

### ISSUE THREAD

**Issue by toliano** [ekumahost/my-complete-high-school-portal#19]
Hi sir. Please, I keep getting this error messages whenever I try to update the studennts profile picture

`AJAX Request Failed
 textStatus=error, errorThrown=`

Many thanks

**Comment by toliano** [comment]
When I checked from the console, I realised `process.php` is missing

`https://ptbackup.poriume.com/control/cpanel/ajax/profile/process.php`

**Comment by kelvin019** [comment]
Hello Toliano,

I have checked our files, we do not have any file like:
/control/cpanel/ajax/profile/process.php

We only have /control/cpanel/ajax/profile/processedit.php

Why changing profile picture from the admin when you can change it from the students dashboard?

**Comment by toliano** [comment]
Oh thanks. I thought it was like the previous version when I uploaded all the students pictures from the admin dashboard.

I tried uploading through the student profile and it shows uploaded but yet the picture was not displaying on the student's account. When I tried to reupload it over again, then I got this error message (link to the image attached) [https://imgur.com/YjQ3zwh](url)

Many thanks for your help
